docs/source/glossary.rst

```diff
@@ -275,7 +275,7 @@ Glossary
 		The source of content for the CDN. Usually a redundant HTTP/1.1 webserver.
 
 	ORT
-		The "Operational Readiness Test" script that stitches the configuration configured in Traffic Portal and generated by Traffic Ops into the :term:`cache servers`.
+		The previous Traffic Control cache config app, replaced by :term:`t3c`.
 
 		.. seealso:: See :ref:`traffic-ops-ort` for a Python implementation of ORT that is (theoretically) compatible with the one actually provided in Apache Traffic Control releases.
```

Comment:
`traffic-ops-ort` label no longer exists - although I think this admonition directive has a few problems

* It mentions ORT.py but doesn't redirect to that section
* It probably shouldn't mention ORT.py anymore since there's no plan to support that in production environments
* It should probably actually be underneath `t3c` since that's what it actually describes/what the link ought to be.